### ISSUE TITLE
[BUG] ERROR TypeError: this.ngZone.runOutsideAngular is not a function

### ISSUE THREAD

**Issue by instantgis** [formio/angular-material-formio#11]
## Environment
Please provide as many details as you can:
node 10.16.0
npm 6.8.0

Everything else versioned as per this repo as is
- Hosting type
  - [x] Form.io
  - [ ] Local deployment
    - Version: 
- Formio.js version:
- Frontend framework:
- Browser:
- Browser version:

## Steps to Reproduce
1. Download this repo & unzip
2. cd to demo folder, npm i
3. ng s

## Expected behavior

Expect this to run out of the box.

## Observed behavior

```
<mat-formio src="https://examples.form.io/kitchensink"></mat-formio>
gives unauthorised - makes sense?
```

this

```
<mat-card>
  <!-- <mat-formio src="https://examples.form.io/kitchensink"></mat-formio> -->
  <mat-formio [form]="form" [submission]="submission"></mat-formio>
</mat-card>
```

bombs out with

```
ng:///AppModule/AppComponent.ngfactory.js:9 ERROR TypeError: this.ngZone.runOutsideAngular is not a function
    at FormioComponent.push../node_modules/angular-formio/FormioBaseComponent.js.FormioBaseComponent.ngOnChanges (vendor.js:167619)
    at checkAndUpdateDirectiveInline (vendor.js:92277)
    at checkAndUpdateNodeInline (vendor.js:103615)
    at checkAndUpdateNode (vendor.js:103554)
    at debugCheckAndUpdateNode (vendor.js:104576)
    at debugCheckDirectivesFn (vendor.js:104519)
    at Object.eval [as updateDirectives] (ng:///AppModule/AppComponent.ngfactory.js:17)
    at Object.debugUpdateDirectives [as updateDirectives] (vendor.js:104507)
    at checkAndUpdateView (vendor.js:103519)
    at callViewAction (vendor.js:103885)

```
## Example
If possible, please provide a screenshot, live example (via [JSFiddle](https://jsfiddle.net/) or similar), and/or example code to help demonstrate the issue.

For code or form JSON, please enclose in a code block:

```js
// your code here
```

**Comment by travist** [comment]
This will be fixed in a new build I am working on. Should be released in the next couple of days.

**Comment by travist** [comment]
Fixed in 1.2.0 build.
